web/static/dlg_repo_edit.js

```diff
@@ -118,15 +114,15 @@ export function show( a_repo_id, a_cb ){
     $(".btn",frame).button();
 
     if ( a_repo_id ){
-        $("#title",frame).on('input', function(){ repoInputChanged(1); });
-        $("#desc",frame).on('input', function(){ repoInputChanged(2); });
-        $("#domain",frame).on('input', function(){ repoInputChanged(4); });
-        $("#capacity",frame).on('input', function(){ repoInputChanged(8); });
-        $("#path",frame).on('input', function(){ repoInputChanged(0x10); });
-        $("#exp_path",frame).on('input', function(){ repoInputChanged(0x20); });
-        $("#pub_key",frame).on('input', function(){ repoInputChanged(0x40); });
-        $("#addr",frame).on('input', function(){ repoInputChanged(0x80); });
-        $("#ep_id",frame).on('input', function(){ repoInputChanged(0x100); });
```

Comment:
This is definately cleaner and more readable getting rid of the magick numbers, though I'm not sure I fully grasp the implications of the change, I'm guessing you were using a bitmask to specify what parts needed to be updated and are now instead just doing a blanket update of the form?

Reply:
Yep, no need to support partial updates.